t/phase.t

```diff
@@ -0,0 +1,273 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
```

Comment:
The tests here (with a few exceptions where I ran into some Perl parsing issues in tests 8 and 9, I'll take a deeper look at those) do test JIT-ability, copying the same logic with regard to examining the output of jit.v (I slimmed down the init_by_lua_block definition since jit.dump never actually runs, but I can add it back for completeness if needed). I also wanted to test the response of the new get_phase() implementation to ensure that we're handing back the same results as we expect from the C API, but if this is unnecessary I will remove it.